omnij-rpc/src/main/groovy/foundation/omni/consensus/OmniCoreConsensusTool.groovy

```diff
@@ -58,25 +58,10 @@ class OmniCoreConsensusTool extends ConsensusTool {
         tool.run(args.toList())
     }
 
-    private SortedMap<Address, ConsensusEntry> getConsensusForCurrency(CurrencyID currencyID) {
-        List<MPBalanceEntry> balances = client.omniGetAllBalancesForId(currencyID)
-
-        TreeMap<Address, ConsensusEntry> map = [:]
-
-        balances.each { MPBalanceEntry item ->
-
-            Address address = item.address
-            ConsensusEntry entry = itemToEntry(item)
-
-            if (address != "" && entry.balance > 0) {
```

Comment:
Similar filters remain for `OmniwalletConsensusTool` and `ChestConsensusTool`. Should we remove those? Could you try that locally?